resources/views/components/menu/account.blade.php

```diff
@@ -27,15 +27,21 @@
 @endguest
 @auth
     <div class="nav-link flex-col justify-center items-end text-2xs" style="line-height: 1.1em">
+        @if($user->RASoftcorePoints && $user->RASoftcorePoints > $user->points_total)
+            <div class='softcore cursor-help' title="Points earned in softcore mode">{{ localized_number($user->RASoftcorePoints) }}</div>
+        @endif
+
         @if($user->points_total)
             <div class="text-color cursor-help" title="Points earned in hardcore mode">{{ localized_number($user->points_total) }}</div>
         @endif
+
         @if($user->points_weighted_total)
             <x-points-weighted-container>
                 <span class='trueratio'>{{ localized_number($user->points_weighted_total) }}</span>
             </x-points-weighted-container>
         @endif
-        @if($user->RASoftcorePoints)
+
+        @if($user->RASoftcorePoints && $user->RASoftcorePoints < $user->points_total)
```

Comment:
This should be `<=`. Softcore points disappear entirely if they're equal to hardcore points.

Reply:
Great catch, resolved in https://github.com/RetroAchievements/RAWeb/pull/1844/commits/d9a4758fca15826acb323b0d356bd837f959f757.